src/main/java/io/visual_regression_tracker/sdk_java/VisualRegressionTrackerConfig.java

```diff
@@ -10,5 +10,5 @@
     private String project;
     private String apiKey;
     private String branchName;
-    private Boolean enableSoftAssert;
+    private boolean softAssert;
```

Comment:
please, keep the old name to be similar to other sdk
in case it needs to be changes we need updated it across all integration libraries